api/GPUCommandEncoder.json

```diff
@@ -117,6 +117,50 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "timestampWrites": {
+          "__compat": {
+            "description": "<code>timestampWrites</code> descriptor property",
+            "spec_url": "https://gpuweb.github.io/gpuweb/#dom-gpucomputepassdescriptor-timestampwrites",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "121",
+                "notes": "Currently supported on ChromeOS, macOS, and Windows only."
+              },
+              "chrome_android": {
+                "version_added": "121"
+              },
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "preview"
+              },
```

Comment:
How did you determine the Safari TP support status of all the newly added features?